### ISSUE TITLE
Add abstract and full paper html rendering

### ISSUE THREAD

**Issue by sylvaticus** [openjournals/joss#879]
Currently only the pdf version of a paper is available, not even the abstract.

However pdfs don't render well on mobiles, and are questionable on any screen type..
It would be much nicer to get a html rendering of the articles with just a "download pdf" for who need to print them.

**Comment by arfon** [comment]
This is something we're definitely interested in doing, but there is a maintenance burden to produce HTML output which is why we moved away from this.

One small point - JOSS papers don't actually have an abstract so we won't be displaying those any time soon :smile:

**Comment by leomrtns** [comment]
I was going to open an issue but my comment is close enough to this one :wink: 
The lack of a summary (wink wink) in text form makes it less inclusive. I am thinking about visual impairment, but PDFs are also suboptimal for other reasons (low bandwith, public/shared PCs or tablets, etc.)

I notice that the RSS does not contain the summary either, which makes the feed much less atractive. Maybe RSS would be easier than HTML with regards to adding the summary (or even the whole text)?

Perhaps offering the original `paper.md` markdown file for download would already be an improvement...

**Comment by arfon** [comment]
Note that we do create JATS XML these days (although we don't link to it from anywhere). Would this be an improvement?

**Comment by leomrtns** [comment]
@arfon for inclusiveness, I don't think so. I am not familiar with JATS XML but I guess you'd need something else to interpret it. XML would otherwise sound like gibberish to a screen reader. By the way I do not use a screen reader, but I remember from discussions that the goal should be HTML or TXT. I mentioned RSS since aggregators render it to HTML.

For other PDF limitations perhaps JATS XML is an alternative, but as I said I am not familiar with it --- which may be an indicator that it is (still) not so common as to be comparable to RSS/HTML...

sorry about the late reply, vacation mode kicking in :smile:

**Comment by danielskatz** [comment]
I think we should consider a text abstract field that we could render on the paper's webpage in addition to having it be in the pdf/jats.  Perhaps this would simply be renaming the Summary section to Abstract, and then working on the processing & display tooling?

Part of my reason for suggesting this is that we really should be submitting an (open) abstract as part of the DOI metadata to make our papers more findable.

**Comment by danielskatz** [comment]
@arfon - Do we have a roadmap of technical changes that we would like to make?

**Comment by arfon** [comment]
> @arfon - Do we have a roadmap of technical changes that we would like to make?

No, but perhaps we could collect a few high priority ones and create some kind of milestone?

**Comment by danielskatz** [comment]
It seems like a good idea.  If we had a list, I could add to it as I look at issues...

**Comment by arfon** [comment]
I made a backlog view here which could be grouped/re-ordered: https://github.com/orgs/openjournals/projects/2/views/1
